pyaedt/icepak.py

```diff
@@ -4118,7 +4114,12 @@ def assign_hollow_block(
 
         if not isinstance(object_name, list):
             object_name = [object_name]
-
+        for o_n in object_name:
+            if self.modeler.get_object_from_name(o_n).solve_inside:
+                self.logger.add_error_message(
+                    "Use ``assign_solid_block`` method with this object as ``solve_inside`` is ``True``."
```

Comment:
```suggestion
                    "Use the  ``assign_solid_block`` method with this object as ``solve_inside`` is ``True``."
```
The "the" appeared in the previous similar message for the ``False`` condition (line 3979).